src/GitHub.Logging/Logging/ILoggerExtensions.cs

```diff
@@ -25,5 +28,37 @@ public static void Assert(this ILogger logger, bool condition, string messageTem
 #pragma warning restore Serilog004
             }
         }
+
+        public static void Time(this ILogger logger, string name, Action method)
+        {
+            var startTime = DateTime.Now;
+            method();
+            logger.Information("{Name} took {Seconds} seconds", name, FormatSeconds(DateTime.Now - startTime));
```

Comment:
Are you sure this should be "Information"? It seems like something somebody would want to give us when debugging, so that makes me feel like it should be "Trace" so it can be turned off.

Reply:
Good catch. 👍